db/rmWrapper.py

```diff
@@ -1214,6 +1221,30 @@ def quests_from_db(self, neLat=None, neLon=None, swLat=None, swLon=None, oNeLat=
                 'task': quest_task, 'quest_template': quest_template})
 
         return questinfo
+        
+    def get_pokestops_changed_since(self, timestamp):
+        query = (
+            "SELECT pokestop_id, latitude, longitude, lure_expiration, name, image, active_fort_modifier "
+            "FROM pokestop "
+            "WHERE DATEDIFF(lure_expiration, '1970-01-01 00:00:00') > 0 AND last_updated >= %s"
```

Comment:
This makes no sense.`lure_expiration` is always "1970-01-01 00:00:00" causing this method retuning dozens to hundrends of stops every 10s, since they all become updated.

Since there's actually no reason to post just stops to a webhook, I'd rather say we change this to something like `get_lures_changed_since()` just to retrieve lure modules. Thus we can add proper checks (see main thread in this PR)

Reply:
It makes sense since it will only compare recently scanned entries and looks for a non default lure_expiration (which is a real date if a lure is active). Sure this can be tweaked but its not as bad?